R/package.R

```diff
@@ -59,17 +59,20 @@ rcmdcheck <- function(path = ".", quiet = FALSE, args = character(),
 
   if (isTRUE(out$timeout)) message("R CMD check timed out")
 
-  res <- parse_check_output(
-    out$result,
+  res <- new_rcmdcheck(
+    stdout = out$result$stdout,
+    timeout = out$result$timeout,
```

Comment:
Here as well, we can keep `stderr`, and maybe also the exit status of the process. `rcmd_safe` returns them.